src/jwt/jwt_utils.rs

```diff
@@ -98,7 +98,7 @@ fn get_input_component(
             html! {
                 <textarea
                     rows="4"
-                    placeholder={"RSA private/public key in PEM (-----BEGIN RSA PRIVATE/PUBLIC KEY-----)"}
+                    placeholder={"RSA private/public key in PEM (-----BEGIN PRIVATE/PUBLIC KEY-----)"}
```

Comment:
still a bad placeholder. here we use the `ECDSA` keys. You can generate such a key and print it to know the label. enter the following command:
```bash
openssl ecparam -name secp256k1 -genkey -noout -out ec-private-key.pem && cat ec-private-key.pem | head -n 1
```
It prints: `-----BEGIN EC PRIVATE KEY-----`

Reply:
additional info of how to generate such keys: https://techdocs.akamai.com/iot-token-access-control/docs/generate-ecdsa-keys